### PR TITLE
create necessary files to run jitsi-meet on the grid 

### PR DESCRIPTION
**Pull request by eyad-hussein**
### Description

Created files for jitsi application in the playground

### Changes

- Frontend files/views needed to for jitsi application
- Edit on existing files to include jitsi

### Related Issues
- https://github.com/codescalersinternships/home/issues/253

### Documentation PR
- https://github.com/threefoldtech/info_grid/pull/631

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/views/solutions_view.vue [hunk 1]
```diff
@@ -144,6 +144,13 @@ export default {
         icon: "wordpress.png",
         route: DashboardRoutes.Applications.Wordpress,
       },
+      {
+        title: "Jitsi",
+        excerpt:
+          "Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features.",
+        icon: "jitsi.png",
```
Comment (zaelgohary) [inline packages/playground/src/views/solutions_view.vue]: Image needs to be changed. @ehab-hassan Can you check this?

File: packages/playground/src/weblets/tf_jitsi.vue [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    await deployGatewayName(grid, selectionDetails.value.domain, {
+      subdomain,
+      ip: vm[0].interfaces[0].ip,
+      port: 80,
+      network: vm[0].interfaces[0].network,
+    });
+
+    finalize(vm);
+  } catch (e) {
+    layout.value.setStatus("deploy", "Rollbacking back due to fail to deploy gateway...");
```
Comment (zaelgohary) [inline packages/playground/src/weblets/tf_jitsi.vue]:
```suggestion
    layout.value.setStatus("deploy", "Rolling back due to failure to deploy gateway...");
```

File: packages/playground/src/views/jitsi_view.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <view-layout>
+    <tf-jitsi />
+
+    <template #list>
+      <TfDeploymentList title="Jitsi Instances" :project-name="name" />
```
Comment (zaelgohary) [inline packages/playground/src/views/jitsi_view.vue]:
```suggestion
      <TfDeploymentList title="Jitsi" :project-name="name" />
```

Just to match other solutions.

**Review by zaelgohary — CHANGES_REQUESTED** [review]
Good job, @eyad-hussein some small comments. Also, please fix failing workflow.

**Comment by zaelgohary** [comment]
While resolving conflicts, you've removed some code. Please, check this.

**Comment by zaelgohary** [comment]
@maxux Can you promote this? https://hub.grid.tf/eyadhussein.3bot/eyadhussein-jitsi-latest.flist
Should be named https://hub.grid.tf/tf-official-apps/jitsi-latest.flist

File: packages/playground/public/info/jitsi.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Jitsi
+---
+
+Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features. For more details, check [Jitsi documentation](https://jitsi.github.io/handbook/docs/intro/).
```
Comment (zaelgohary) [inline packages/playground/public/info/jitsi.md]: The link should redirect to app manual link. The link should work once info_grid PR is merged.
```suggestion
Jitsi Meet is a set of Open Source projects which empower users to use and deploy video conferencing platforms with state-of-the-art video quality and features. For more details, check [Jitsi documentation](https://www.manual.grid.tf/documentation/dashboard/solutions/jitsi.html).
```

Comment (eyad-hussein) [reply packages/playground/public/info/jitsi.md]: done, applied suggestion

**Comment by maxux** [comment]
> @maxux Can you promote this? https://hub.grid.tf/eyadhussein.3bot/eyadhussein-jitsi-latest.flist Should be named https://hub.grid.tf/tf-official-apps/jitsi-latest.flist

Promoted :)
https://hub.grid.tf/tf-official-apps/jitsi-latest.flist.md

File: packages/playground/src/weblets/tf_jitsi.vue [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+const flist: Flist = {
+  value: "https://hub.grid.tf/tf-official-apps/jitsi-latest.flist",
+  entryPoint: "/sbin/zinit init",
+};
+const dedicated = ref(false);
+const certified = ref(false);
+const ipv4 = ref(false);
+const ipv6 = ref(false);
+const mycelium = ref(true);
+const planetary = ref(true);
```
Comment (Mahmoud-Emad) [inline packages/playground/src/weblets/tf_jitsi.vue]: Based on https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3287 we should enable only the `mycelium` network

**Review by Mahmoud-Emad — CHANGES_REQUESTED** [review]
Good job ya Eyad!

**Comment by Mahmoud-Emad** [comment]
@eyad-hussein 
Please resolve the conflicts

File: packages/playground/src/weblets/tf_jitsi.vue (hunk 5 above)
Comment (eyad-hussein) [reply packages/playground/src/weblets/tf_jitsi.vue]: Done, only mycelium is enabled by default

**Comment by Mahmoud-Emad** [comment]
@eyad-hussein is the PR ready?

**Comment by eyad-hussein** [comment]
I believe it lacks the right jitsi logo https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3346#discussion_r1735166609. Everything else is fine. @Mahmoud-Emad

**Comment by ehab-hassan** [comment]
use this icon ll be better
jitsi

File: packages/playground/src/views/solutions_view.vue (hunk 1 above)
Comment (eyad-hussein) [reply packages/playground/src/views/solutions_view.vue]: The logo is changed with https://github.com/threefoldtech/tfgrid-sdk-ts/pull/3346#issuecomment-2328599110

**Review by zaelgohary — APPROVED** [review]
Good job! @eyad-hussein 

image

image

**Comment by Mahmoud-Emad** [comment]
LGTM, Good job ya Eyad

image

**Review by Mahmoud-Emad — APPROVED** [review]

**Pull request merged**